server/pulp/server/webservices/views/consumer_groups.py

```diff
@@ -56,6 +57,7 @@ def delete(self, request, consumer_group_id):
         result = manager.delete_consumer_group(consumer_group_id)
         return generate_json_response(result)
 
+    @must_have_json_body
```

Comment:
Please reorder to make the `auth_required` statements always the top-most decorator. If a request isn't authorized I don't want to tell the user anything until they come back with proper authorization. The way this is written will have auth_required will have validation occur before authentication+authorization.